### PR TITLE
QuickPanel - Live Preview

### PR DESCRIPTION
**Pull request by Enteleform**
Hi, not sure if I'm doing this right ( new to Git / GitHub ) but here's an extension I wrote last night for AlignTab.  It's pretty straightforward & is documented with headers from a plugin I'm working on ( should be up soon as "Parkour", best viewed without wrap ).

Feel free to send some feedback my way, just getting started with coding : )

{
	"keys": ["ctrl+super+/"],
	"command": "align_tab_quick_panel",
},

AlignTabQuickPanel

### CODE REVIEW

File: quickpanel.py [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+	#▐▌»»▒▐▌──────────────────────────────────────────────────────────────────▐▌▒««▐▌‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾‾¦•⌡#
+
+	def quickpanel_On_Done ( self, index ):
+
+		active_view            = AlignTabQuickPanel.run.active_view
+		i_quickpanel_selection = index
+
+		if i_quickpanel_selection > 0:
+
+			active_view.run_command ( "expand_selection_to_paragraph" )
```
Comment (FiCode) [inline quickpanel.py]: Added Expand Text @ On_Done / Select Pattern

**Comment by FiCode** [comment]
Also, Here's a GIF of the extension's functionality:

[[ Sublime Text ] AlignTab QuickPanel Demo [2015-05-24]](http://i.imgur.com/Vfrrjhi.gif)

**Comment by randy3k** [comment]
Thanks for the contribution. It is great idea. But I think I rather implement it slightly differently for consistency.

**Comment by FiCode** [comment]
No worries.  May I ask what you mean?

Like I mentioned, this is my first pull request & I'd like to increase the quality of my contributions.

**Pull request closed without merging**